nni/algorithms/compression/v2/pytorch/pruning/tools/metrics_calculator.py

```diff
@@ -120,7 +120,7 @@ def calculate_metrics(self, data: Dict[str, Tensor]) -> Dict[str, Tensor]:
 
             metric = torch.ones(*reorder_tensor.size()[:len(keeped_dim)], device=reorder_tensor.device)
             across_dim = list(range(len(keeped_dim), len(reorder_dim)))
-            idxs = metric.nonzero()
+            idxs = metric.nonzero(as_tuple=False)
```

Comment:
the default value of `as_tuple` is False, so why as it?

Reply:
because if don't set `as_tuple`, a warning will appear to ask for setting `as_tuple` value.